riscv/src/compiler.rs

```diff
@@ -137,18 +137,6 @@ pub fn compile(
     // for compilation, and will not be called.
     statements = replace_coprocessor_stubs(statements, coprocessors).collect::<Vec<_>>();
 
-    // Sort the objects according to the order of the names in object_order.
```

Comment:
Could you add an assertion checking that the sum of the sizes is less than 2**31 or something?

Reply:
Ah sorry, this is not needed anymore.